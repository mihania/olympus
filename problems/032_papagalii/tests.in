20
2 1
3 3
4 5
5 7
6 9
7 11
8 13
9 15
10 17
2 19
3 21
4 23
5 25
6 27
7 29
8 31
9 33
10 35
5 37
10 40