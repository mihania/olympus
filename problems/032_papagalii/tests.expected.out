2
55
7084
2330445
1478314266
1547847846090
2426870706415800
5332692942907262361
15654733143626084944150
6564120420
4048514844039120
357557785658996609700
14828221541905870299371280
436492157579951023496648300136
10971665040700591613884634129449456
258864862352136696653733497266326332280
6042775330755648746586154003846346316488175
143934427865598739459773469503215339354507434714
92249492826218529886086900321774895545
1357740187499993272317100164273173056232088998671872900
